chimney/src/test/scala/io/scalaland/chimney/DslFSpec.scala

```diff
@@ -726,5 +726,53 @@ object DslFSpec extends TestSuite {
         }
       }
     }
+
+    "implicit conflict resolution" - {
+
+      final case class InnerIn(value: String)
+      final case class InnerOut(value: String)
+
+      final case class OuterIn(value: InnerIn)
+      final case class OuterOut(value: InnerOut)
+
+      implicit val pureTransformer: Transformer[InnerIn, InnerOut] =
+        in => InnerOut(in.value)
```

Comment:
This implementation is equivalent to derived version. For testing maybe better to add some prefix `s"pure: ${in.value}"`?